docs/products/mysql/howto/enable-slow-queries.rst

```diff
@@ -24,6 +25,11 @@ Follow these steps to enable your slow queries in your Aiven for MySQL service v
 
 Your Aiven for MySQL service can now log slow queries. If you want to simulate slow queries to check this feature, check the next section.
 
+.. warning::
+
+    Aiven managed MySQL services use table output for the slow query log.
+    This means that the slow query log will not function on a read-only replica.
```

Comment:
Let's simplify it a little:
The  slow query logging is not supported on read-only replicas because Aiven for MySQL services use table output for the slow query log.